Future-Hope/Future-Hope/Controllers/AppDelegate.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  AppDelegate.swift
+//  Future-Hope
+//
+//  Created by Hector Steven on 8/15/19.
+//  Copyright © 2019 Hector Steven. All rights reserved.
+//
+
+import UIKit
+import Firebase
+import GoogleSignIn
+import FBSDKCoreKit
+
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+	var window: UIWindow?
+
+
+	func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+		FirebaseApp.configure()
+		GIDSignIn.sharedInstance()?.clientID = FirebaseApp.app()?.options.clientID
+		GIDSignIn.sharedInstance()?.delegate = self
+		//FBSDK
+//		ApplicationDelegate.shared.application(application, didFinishLaunchingWithOptions: launchOptions)
```

Comment:
I'm not sure what this line is. If you're not using it I would take it out to clean up the code. If you are still going to use it, what does it do?

Reply:
It launches the facebook permission Alert .. I think.. It was in docs and not sure if it is required.

Reply:
Okay, good to know.